nodes/pods/run_once_duration_override/run-once-duration-override-release-notes.adoc

```diff
@@ -29,3 +29,25 @@ The following advisory is available for the {run-once-operator} 1.0.0:
 * This is the initial, generally available release of the {run-once-operator}. For installation information, see xref:../../../nodes/pods/run_once_duration_override/run-once-duration-override-install.adoc#run-once-duration-override-install[Installing the {run-once-operator}].
 
 // No bug fixes or known issues to list
+[id="run-once-duration-override-operator-release-notes-1-0-1"]
+== OpenShift {run-once-operator} 1.0.1
```

Comment:
I know it's like this in the 1.1.0 one, but it's wrong. Can you remove the "OpenShift" in front of the operator name here and on the 1.1.0 heading on line 18 too?

Reply:
Done!